src/components/FullScreenDropZone.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import styled from 'styled-components';
+
+const DropDiv = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+`;
+
+const FullScreenDropZone = ({
+  children,
+  noClick,
+  closeModal,
+  showModal,
+}) => {
+  return (
+    <>
```

Comment:
[change]
No need to add an extra fragment if it just has one child. Fragments are supposed to be used only where there are multiple root nodes in JSX.